it/mail/chrome/messenger/subscribe.dtd

```diff
@@ -35,10 +35,10 @@
 
  ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY subscribeDialog.title        "Sottoscrizioni">
-<!ENTITY subscribeButton.label        "Sottoscrivi">
+<!ENTITY subscribeDialog.title        "Iscriviti">
```

Comment:
CORRECTIONS NEEDED. 

I would change "Iscriviti" with "Iscrizioni", to keep the same difference there was between the original sentences 38 and 39 (Sottoscrizioni-Sottoscrivi --> Iscrizioni-Iscriviti).